include/glow/Backends/DeviceManager.h

```diff
@@ -27,12 +27,11 @@
 #include <string>
 
 namespace glow {
+namespace runtime {
 
 /// Callback signalling success/failure of loading a Module onto a device.
 using ReadyCBTy = std::function<void(const Module *, runtime::ResultCode)>;
```

Comment:
why not move this to runtime types too?

Reply:
We can, it's only used in the deviceManagers (and the test deviceManager in executor test) So it decided to leave it in DeviceManger.h.